index.html

```diff
@@ -222,103 +230,116 @@ <h1>Multi-Post
+                            <button type="button" class="resume">Get my resume</button>
+                        </div>
+                    </div>
+                    <div class="languages">
+                        <h1 class="lang">Languages</h1>
 
-                    <img class="icons" src="./images/About My Self.jpg" alt="">
+                        <img class="icons" src="./images/About My Self.jpg" alt="">
 
-                    <div class="comp" id="color">
-                        <ul>
-                            <li><img class="icon" id="icon" src="./images/Frame 9.png" alt="">
-                            <li> <img class="icon" id="icon" src="./images/Frame 10.png" alt=""></li>
-                            <li> <img class="icon" id="icon" src="./images/Frame 11.png" alt=""></li>
+                        <div class="comp" id="color">
+                            <ul>
+                                <li><img class="icon" id="icon" src="./images/Frame 9.png" alt=""></li>
+                                <li> <img class="icon" id="icon" src="./images/Frame 10.png" alt=""></li>
+                                <li> <img class="icon" id="icon" src="./images/Frame 11.png" alt=""></li>
 
+                                <li><img class="icon2" id="iconjs"  src="./images/languages/Frame 9.png" alt=""></li>
+                                <li> <img class="icon2" id="iconhtml" src="./images/languages/Frame 10.png" alt=""></li>
+                                <li> <img class="icon2" id="iconcss" src="./images/languages/Frame 11.png" alt=""></li>
+
+                        </div>
+
+
+                        <div class="comp">
+                            <li class="frame">Frameworks</li>
+                            <img class="foward" src="./images/Screenshot from 2022-09-24 10-08-18.png" alt="">
+                            <hr id="hr1">
+                            <li class="frame">Skills</li>
+                            <img class="foward" src="./images/Screenshot from 2022-09-24 10-08-18.png" alt="">
+                            <hr id="hr2">
+                            </ul>
+                        </div>
                     </div>
-                    <div class="comp">
-                        <li class="frame">Frameworks</li>
-                        <img class="foward" src="./images/Screenshot from 2022-09-24 10-08-18.png" alt="">
-                        <hr style="height:1px;border-width:0;color:gray;background-color:gray">
-                        <li class="frame">Skills</li>
-                        <img class="foward" src="./images/Screenshot from 2022-09-24 10-08-18.png" alt="">
-                        <hr style="height:1px;border-width:0;color:rgb(40, 38, 165);background-color:rgb(59, 32, 212)">
-                        </ul>
-                    </div>
+
                 </div>
 
                 <div class="contact-me">
 
                     <h1 class="contat">Contact me</h1>
```

Comment:
- [ ] [REQUIRED] Good job so far! I realized that you need to work a little more on the `contact section`, please, it shows that the location of the elements in the `contact section` is broken, so I tried to `highlight` the `header` element, `p` element, `inputs` and `button` element in the picture below.

![image](https://user-images.githubusercontent.com/63915024/194588746-d2b57924-d0ff-42fa-a8fe-ce9323e2ba94.png)

Reply:
- [ ] Please try to fix the issue adressed by the previous reviewer. The reason that you don't see as same as the previous reviewer is your screen size is smaller and you might applied fixed widths to elements rather than positioning them with flexbox in your mobile project. Here's what I also see: 
![image](https://user-images.githubusercontent.com/91486791/194619780-af9eb458-3a45-41bd-ab90-5d2d0d080a56.png)
- [ ] Also, I recommend fix blurry background image. You should use background-color and border-radius attributes to arrange this background color and shape.